backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc

```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-about-kopia"]
+= About Kopia
+include::_attributes/common-attributes.adoc[]
+:context: oadp-about-kopia
+
+toc::[]
+
+Kopia is a fast and secure open-source backup and restore tool that allows you to create encrypted snapshots of your data and save the snapshots to remote or cloud storage of your choice.
+
+Kopia supports network and local storage locations, as well as many cloud or remote storage locations, including:
+
+* Amazon S3 and any cloud storage that is compatible with S3
+* Azure Blob Storage
+* Google Cloud Storage Platform
+
+Kopia uses content-addressable storage for snapshots:
+
+* Each snapshot is always incremental. This means that all data is uploaded once to the repository based on file content, and a file is only uploaded to the repository again if it is modified. 
+
+* Multiple copies of the same file are stored once, meaning deduplication. After moving or renaming large files, Kopia can recognize that they have the same content and does not upload them again.
+
+
+[id="oadp-kopia-integration"]
+== OADP integration with Kopia
+
+OADP 1.3 supports Kopia as the backup mechanism for pod volume backup in addition to Restic. You must choose one or the other at installation by setting the `uploaderType` field in the `DataProtectionApplication` custom resource (CR). The possible values are `restic` or `kopia`. If you do not specify an  `uploaderType`, OADP 1.3 defaults to using Kopia as the backup mechanism. The data is written to and read from a unified repository.
+
+.`DataProtectionApplication` configuration for Kopia
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: dpa-sample
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+#...
```

Comment:
Per [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks):
```suggestion
# ...
```